azkaban-web-server/src/main/java/azkaban/scheduler/QuartzScheduler.java

```diff
@@ -113,27 +116,26 @@ public void unregisterJob(final String groupName) throws SchedulerException {
    *
    * @param cronExpression the cron schedule for this job
    * @param jobDescription Regarding QuartzJobDescription#groupName, in order to guarantee no
-   * duplicate quartz schedules, we design the naming convention depending on use cases:
-   * <ul>
-   *   <li>User flow schedule: we use {@link org.quartz.JobKey#JobKey} to represent the identity
-   *   of a flow's schedule. The format follows "$projectID_$flowName" to guarantee no duplicates.
-   *   </li>
-   *   <li>Quartz schedule for AZ internal use: the groupName should start with letters, rather
-   *   than number, which is the first case. </li>
-   * <ul>
+   * duplicate quartz schedules, we design the naming convention depending on use cases: <ul>
+   * <li>User flow schedule: we use {@link org.quartz.JobKey#JobKey} to represent the identity of a
+   * flow's schedule. The format follows "$projectID_$flowName" to guarantee no duplicates. </li>
+   * <li>Quartz schedule for AZ internal use: the groupName should start with letters, rather than
+   * number, which is the first case. </li> <ul>
```

Comment:
Looks like this got messed up by Save Actions. Maybe @HappyRay can remember how to prevent this from happening.

Reply:
Please see the dev guide.

Reply:
Intellij should be able to recognize the proper JavaDoc syntax.

Reply:
Paragraphs, as in `<p>`? There is no `<p>` here. I believe my style config is up-to-date.. But could you try this on your IDEA?